pyproject.toml

```diff
@@ -50,6 +50,7 @@ test = [
     "joblib",
     "jacobi",
     "matplotlib",
+    "numpy<1.24",
```

Comment:
A question out of curiosity only, why is the upper bound on numpy needed here? I haven't been paying attention to NumPy releases lately, so are there some changes that we should be paying attention to?

Reply:
It is only required for testing and temporary, because I use numba in the tests and current numba complains when you have numpy >= 1.24.